src/components/v2v3/V2V3Project/V2V3ManageTokensSection/V2RedeemModal.tsx

```diff
@@ -259,7 +261,15 @@ export default function V2RedeemModal({
                 disabled={totalBalance?.eq(0)}
                 onChange={val => setRedeemAmount(val)}
               />
+              {tokenSymbol && tokenAddress && (
```

Comment:
**Nit-pick:** We decided a while ago to stick with `{ condition ? <component/> : null }` in cases like this, rather than just `&&`. Sometimes the `&&` can have unintended consequences such as displaying `0` in the UI

Reply:
Sure! Didn't know there are consequences with approach below. nice learn, thanks!